scripts/ubuntu/mongo-orchestration-config.json

```diff
@@ -1,5 +1,6 @@
 {
 	"releases": {
+		"40-release": "/home/vagrant/4.0/usr/bin",
 		"36-release": "/home/vagrant/3.6/usr/bin",
```

Comment:
I don't think anything is using this. It was only important as the first version listed under "releases", which made it the default. Now that "40-release" is here, I think we can just remove this.

Reply:
We'll leave it for now.